src/GlobalStyle.tsx

```diff
@@ -8,18 +8,20 @@ const GlobalStyle = createGlobalStyle`
   body {
     min-height: 100%;
     height: 100%;
-    font-family: "Helvetica Neue";
+    font-family: "Helvetica Neue", Sans-Serif;
     text-decoration: none;
     color: ${theme.colors.athensGray};
     background: ${theme.colors.black};
+    padding: 24px;
```

Comment:
I'd probably suggest using `${theme.space[6]}` here

Reply:
yeee this is nice but weirdly it doesn't work for me :(

<img width="477" alt="Screenshot 2020-09-22 at 18 59 30" src="https://user-images.githubusercontent.com/53219789/93914011-3049de80-fd06-11ea-99ab-e5821572d594.png">

Reply:
Ah yeah, `padding: ${theme.space[6]}px;`